coding/python_style_guide.rst

```diff
@@ -12,648 +13,671 @@ To request changes to these standards, please file an :ref:`RFC <decision-making
 
-Imports should be grouped in the following order, with each group separated by a blank line:
+- assignment (``=``),
+- augmented assignment (``+=``, ``-=``, etc.),
+- comparisons (``==``, ``<``, ``>``, ``!=``, ``<>``, ``<=``, ``>=``, ``in``, ``not in``, ``is``, ``is not``),
+- Booleans (``and``, ``or``, ``not``).
 
-1. standard library imports
-2. related third party imports
-3. local application/library specific imports
+Use spaces around these arithmetic operators:
 
-When importing a class from a class-containing module,
-it's usually okay to do this:
+- addition (``+``),
+- subtraction (``-``)
 
-.. code-block:: py
+Never surround these binary arithmetic operators with whitespace:
 
-   from myclass import MyClass
-   from foo.bar.yourclass import YourClass
+- multiplication (``*``),
+- division (``/``),
+- exponentiation (``**``),
+- floor division (``//``),
+- modulus (``%``).
 
-But if that causes local name clashes, then do this instead: 
+For example:
 
 .. code-block:: py
 
-   import myclass
-   import foo.bar.yourclass
+   i = i + 1
+   submitted += 1
+   x = x*2 - 1
+   hypot2 = x*x + y*y
+   c = (a + b)*(a - b)
 
-and use ``myclass.MyClass`` and ``foo.bar.yourclass.YourClass``. 
+This deviates from PEP 8, which `allows whitespace around these arithmetic operators if they appear alone <https://www.python.org/dev/peps/pep-0008/#id28>`__.
+Error codes: N226 and N228.
 
-Relative imports SHOULD be used when importing another module from the same package
-Consider this layout: 
+.. _style-guide-py-multiline-assignment-whitespace:
 
-.. code-block:: text
+Keyword assignment operators SHOULD be surrounded by a space when statements appear on multiple lines
+-----------------------------------------------------------------------------------------------------
 
-   mypkg/
-       __init__.py
-       foo.py
-       bar.py
+However, if keyword assignments occur on a single line, where should be no additional spaces.
 
-If ``foo`` wants to import ``bar``, the safe way to do this (Python 2.6 and later) is to use relative import:
+Thus this: 
 
 .. code-block:: py
 
-   from . import bar
+   # whitespace around multi-line assignment
+   funcA(
+       karg1 = value1,
+       karg2 = value2,
+       karg3 = value3,
+   )
+
+   # no whitespace around single-line assigment
+   funcB(x, y, z, karg1=value1, karg2=value2, karg3=value3)
 
-Or, if you just want a few symbols from bar, this also works:
+Not this: 
 
 .. code-block:: py
 
-   from .bar import thing1, thing2
+   funcA(
+       karg1=value1,
+       karg2=value2,
+       karg3=value3,
+   )
 
-This avoids any danger of name collision with a module on the python path named bar.
-Relative import statements are richer than suggested by the example; see :pep:`328` for details.
+   aFunction(x, y, z, karg1 = value1, karg2 = value2, karg3 = value3)
 
-.. _style-guide-py-4-5:
+`Opposes PEP 8 <https://www.python.org/dev/peps/pep-0008/#id28>`__.
+Error code: N251.
```

Comment:
Good catch. I realize I was originally inconsistent with `N251` in my ignore lists, oops.

So now autoflake8 will ignore fixing `N251`, but flake8 *will* lint it. Let's flag `N251` for another RFC because that multi-line keyword arg assignment format is indefensible. :)